### DISCUSSION TITLE
Ive tried everything RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory

### DISCUSSION THREAD

**Discussion by Lucyan11** [neonbjb/tortoise-tts#254] [Q&A]
It was working great, then my power went out during a sess and poof, this error.  Ive tried EVERYTHING except he right thing, please help

windows 10, followed instructions, was working now its not

(voice) D:\VoiceSynth\tortoise-tts>python tortoise/do_tts.py --text "I'm going to speak this" --voice random --preset fast
Traceback (most recent call last):
  File "tortoise/do_tts.py", line 27, in <module>
    tts = TextToSpeech(models_dir=args.model_dir)
  File "D:\VoiceSynth\tortoise-tts\tortoise\api.py", line 236, in __init__
    self.diffusion.load_state_dict(torch.load(get_model_path('diffusion_decoder.pth', models_dir)))
  File "C:\Users\c_law\Anaconda3\envs\voice\lib\site-packages\torch\serialization.py", line 777, in load
    with _open_zipfile_reader(opened_file) as opened_zipfile:
  File "C:\Users\c_law\Anaconda3\envs\voice\lib\site-packages\torch\serialization.py", line 282, in __init__
    super(_open_zipfile_reader, self).__init__(torch._C.PyTorchFileReader(name_or_buffer))
RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory

**Comment by neonbjb** [comment]
Your `diffusion_decoder.pth` file is corrupt. Clear out ~/.cache/tortoise.

**Comment by Lohnii** [comment]
This was happening to me as well and I got rid of it by downloading the models manually (you can find the urls in api.py) and moving them to C:\Users\{YOUR USER}\.cache\tortoise\models

**Reply by manmay-nakhashi** [reply]
@Lohnii error might be coming in wav2vec model can you check ? , you might delete it and download that again.

**Reply by Lohnii** [reply]
I don't know which model it came from but I have a problem with huggingface where I get an error when i try to download large models from the command line, so i downloaded them manually and that fixed it, thanks!
